ui/components/multi-select-wrapper.js

```diff
@@ -179,6 +182,10 @@ const MultiSelectWrapper = (props) => {
       ...base,
       backgroundColor: base.backgroundColor2,
     }),
+    input: (base) => ({
+      ...base,
+      color: theme.palette.type === 'dark' ? darkTheme.palette.secondary.white : '#141414',
```

Comment:
Thankyou for making sure we are using theme color, I wonder if we can also have theme color for `#141414` or any near by color

Reply:
I found a color similar to that from theme and i just pushed the code. Thanks.